docs/panel/media-library.md

```diff
@@ -27,4 +27,7 @@ In order to use images from Unsplash - you need to download them to your library
 
 Media library is accessible via `/api/v1/media/` endpoints. Visit your API docs to learn more.
 
+!!! note
+    See also: [Working with media](/docs/API/media-library/)
```

Comment:
As previous comment, this should be for example Working with media using API etc.